Update enabling delivery api (#6670)
umbraco/UmbracoDocs: 15/umbraco-cms/reference/content-delivery-api/README.md

### Enable the Content Delivery API

> You can enable the Delivery API when creating you project initially by using the `--use-delivery-api` or `-da` flag. This will automatically add the necessary configuration to your project.

Reviewer: ```suggestion
When creating your project, you can enable the Delivery API using the `--use-delivery-api` or `-da` flag. This will automatically add the necessary configuration to your project.
```

```bash
dotnet new umbraco -n MyProject -da
```

> Yuo can also enable it after the fact with the following steps:

Reviewer: ```suggestion
You can also enable the Delivery API at a later point by following these steps:
```
